pykickstart/commands/timesource.py

```diff
@@ -0,0 +1,146 @@
+#
+# Martin Kolman <mkolman@redhat.com>
```

Comment:
Drop emails from copyright messages, we can use git blame to figure out who wrote things.

Reply:
Removed.